{"name": "test_create_with_invalid_data[book_data0]", "status": "passed", "description": "Проверяем, что нельзя создать книгу с некорректными данными, нарушиющих типы", "parameters": [{"name": "book_data", "value": "(2, 'asd')"}], "start": 1676241356482, "stop": 1676241356491, "uuid": "335c95f3-6c08-4595-a31e-4b0ec281ef59", "historyId": "b6b684fca315a9b10e94241db5295068", "testCaseId": "0d2a4e16b7c113dbdc5907d1b75a2e47", "fullName": "test_API#test_create_with_invalid_data", "labels": [{"name": "tag", "value": "negative"}, {"name": "suite", "value": "test_API"}, {"name": "host", "value": "jenkins"}, {"name": "thread", "value": "773-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_API"}]}